Saucedemo/Core/Wrappers/Error.cs

```diff
@@ -0,0 +1,25 @@
+﻿using OpenQA.Selenium;
+
+namespace Saucedemo.Core.Wrappers
+{
+    public class Error
```

Comment:
it is not entirely clear why the element is called an error.
ErrorElement would be better already